src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "AvenirBook";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.form1 {
+  margin-top: 20px;
+}
```

Comment:
the styles are identical, so for another form you can apply this class `form` too and remove `form1`